sdk/pom.xml

```diff
@@ -247,15 +242,15 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.12.4</version>
 				<configuration>
-					<skipTests>false</skipTests>
+					<skipTests>true</skipTests>
```

Comment:
Why not to remove this part at all. So all external users will package SDK with tests by default - it's safer.
We can cancel tests running in Intellij maven view or by hand if cmd is used.